lib/rouge/lexers/msgtrans.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class MsgTrans < RegexLexer
+      title "MessageTrans"
+      desc "RISC OS message translator messages file"
+      tag 'msgtrans'
+      filenames 'Messages*'
+
+      state :root do
+        rule %r/^#[^\n]*/, Comment
+        rule %r/[^\n ,):?\/]+/, Name::Variable
+        rule %r/[\n\/?]/, Operator
```

Comment:
A newline is really considered an operator? There aren't other operators?

Reply:
Well, `?` is actually a wildcard. `/` is an "or" operator, and newline is also considered as an "or" operator if the preceding line didn't contain a result value (i.e. it didn't contain a `:` separator). Since you can't really colour a newline, I didn't think it was worth bothering to distinguish between cases where they separate different logical lines and where they act as operators.

There's really very little scope for extending the syntax with any further operators - only `,` `)` and `%` are reserved characters (maybe `\t` too).

Reply:
Hmmm, yeah, your approach with newlines makes sense. 

As for `?`, I'm not sure what that is best tokenised as. I'm happy to leave it as `Operator`.